Modules/vc-module-marketing/VirtoCommerce.MarketingModule.Data/ExportImport/MarketingExportImport.cs

```diff
@@ -26,7 +26,11 @@ public class MarketingExportImport
         private readonly ICouponService _couponService;
         private readonly IPromotionUsageService _promotionUsageService;
 
-        public MarketingExportImport(JsonSerializer jsonSerializer, IPromotionSearchService promotionSearchService, IDynamicContentSearchService dynamicContentSearchService, IPromotionService promotionService, IDynamicContentService dynamicContentService, ICouponService couponService, IPromotionUsageService promotionUsageService)
+        public MarketingExportImport(
```

Comment:
Please remove these changes from this class, all of them not have sense in 3.0 version.

Reply:
done